src/libs/elektra/mount.c

```diff
@@ -358,7 +358,7 @@ KeySet * elektraDefaultGlobalConfig (void)
 
 	// TODO: this is a poor way of detecting whether cache is compiled, but simply
 	// matching against cache might fail because of other plugins (e.g. "cachefilter")
-	if (strstr (ELEKTRA_PLUGINS, ";cache;") != NULL)
+	if (strstr (ELEKTRA_PLUGINS, ";cache;") != NULL && ksLookupByName (keys, "system/elektra/cache/enabled", 0))
```

Comment:
I would remove the strstr part as it might be wrong (external compilation of cache plugin or similar). But this can be done in another PR.